client/strings/de.json

```diff
@@ -559,9 +559,9 @@
   "ToastRSSFeedCloseSuccess": "RSS-Feed geschlossen",
   "ToastSessionDeleteFailed": "Sitzung konnte nicht gelöscht werden",
   "ToastSessionDeleteSuccess": "Sitzung gelöscht",
-  "ToastSocketConnected": "Socket connected",
-  "ToastSocketDisconnected": "Socket disconnected",
-  "ToastSocketFailedToConnect": "Socket failed to connect",
+  "ToastSocketConnected": "Server online",
+  "ToastSocketDisconnected": "Server offline",
```

Comment:
This isn't correct because the server can be online but the web socket connection is not working.
I don't think there would be a translation for Web Socket but maybe you have a translation for connected/disconnected?

Reply:
I change it ;)

Reply:
Ok I've built a mess with the sync of my fork. Can you change that?

SocketConnected: Verbindung zum Server hergestellt
SocketDisconnected: Verbindung zum Server verloren